queryscript/src/compile/coerce.rs

```diff
@@ -21,9 +21,12 @@ macro_rules! must_atomic {
     };
 }
 
+#[derive(Clone, Debug)]
 pub enum CoerceOp {
     Binary(BinaryOperator),
+    #[allow(unused)]
```

Comment:
I think these are both used?

Reply:
Not according to the rust compiler.  I'm unsure why it started complaining after my change, but they don't appear to be constructed anywhere.

Reply:
Ah interesting, okay sounds good